src/client/todos/todo.styl

```diff
@@ -0,0 +1,26 @@
+.todo-item
+  font-size .875em
+  margin-bottom .25em
+  max-width 42em
+
+  &__label, &__button
```

Comment:
Thank you for a video, I will have to take a look at it. Hard times for OSS maintainer, when commiters seems to be smarted than me :-)